pkg/resolver/search.go

```diff
@@ -45,14 +45,16 @@ func Search(ctx context.Context, input []*model.SearchInput) ([]*SearchResult, e
 	return srchResult, nil
 }
 
+func (s *SearchResult) KeywordSearch() { //need this function and resolver for tests.
```

Comment:
Why do we need a different function?  A search query could include both keywords and filters so we need to process from the same resolver.

Reply:
This was supposed to be removed. Thanks for catching that.